src/form/input/Input.tsx

```diff
@@ -49,7 +49,6 @@ export type InputProps = Omit<
   isDisabled?: boolean;
   hasError?: boolean;
   customClassName?: string;
-  inputContainerRef?: React.RefObject<HTMLDivElement>;
```

Comment:
This will be a breaking change. Let's make sure to include it in the release notes. cc: @mucahit